sklearn_porter/Porter.py

```diff
@@ -57,10 +57,40 @@ def __init__(self, estimator, language='java', method='predict', **kwargs):
 
         # Determine the local version of sklearn:
         from sklearn import __version__ as sklearn_ver
+
+        ## Suffix lists for minor, patch version in sklearn
+        MINOR_VERSION_SUFFIX_LIST = ["rc3", "rc2", "rc1", "b1", "a1"]
+        PATCH_VERSION_SUFFIX_LIST = ["post1", "0b1", "0b2"]
```

Comment:
At least post1 is not part of the patch version, but after the patch version:
![Uploading grafik.png…]()